src/game_api/entity.cpp

```diff
@@ -716,3 +716,16 @@ void Entity::set_pre_collision2(std::uint32_t reserved_callback_id, std::functio
     }
     hook_info.pre_collision2.push_back({reserved_callback_id, std::move(pre_collision2)});
 }
+
+std::span<uint32_t> Entity::get_items()
+{
+    if (items.size)
+        return std::span<uint32_t>(items.uids().begin(), items.uids().end());
+
+    return {};
+}
+
+void Movable::light_on_fire_broken()
+{
+    this->light_on_fire(0x64);

```

Comment:
Why exactly `0x64`, maybe put the answer in a comment?

Reply:
The game normally lights you up for 100 frames, this is hard coded value.

I guess there could be some sources that set it to different times, but. This is a broken version of this function that shouldn't be used, kept for backwards compatibility. It doesn't show in the doc

Reply:
Sorry for the confusion, I got why. My point was just that it's better to put a comment for future us when we're not currently looking at a PR. Magic numbers are not cool.